stable/aws-for-fluent-bit/values.yaml

```diff
@@ -141,6 +164,75 @@ elasticsearch:
   # extraOutputs: |
   #   Index = my-index
 
+s3:
+  enabled: true
+  match: "*"
+  bucket:
+  region: "us-east-1"
+  jsonDateKey: "date"
+  jsonDateFormat: "iso8601"
+  totalFileSize: "100M"
+  uploadChunkSize: 5M
```

Comment:
why are the other in quotes but not this one?

Reply:
updated